code/modules/projectiles/projectile/bullets.dm

```diff
@@ -232,3 +232,6 @@
 	if(isalien(target))
 		return 0
 	..() // Execute the rest of the code.
+
+/obj/item/projectile/bullet/a762

```

Comment:
...Might have actually figured out why this was, and I feel like a moron for it.
Hang on while it compiles.